src/tests/Mission.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import MissionsList from '../components/MissionsList';
+import store from '../redux/store';
+
+it('matches snapshot', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <MissionsList />
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] You did a great job adding these snapshots, but as per project instructions, kindly test all the components and functions. You can test to see if certain elements such as a title, button, or text are on the page(browser).

![test](https://github.com/Rifat8080/react-redux-group-project/assets/53124548/33a87037-66e1-43d4-bf81-399af58fb18b)

Reply:
@elisha2kyakpo1 Can you clear that for me, we tested all the components as a group of two persons we only have three tests the Rockets.js, Missions.js and MyProfile.js